sample-app/src/main/java/com/joinforage/android/example/pos/receipts/templates/txs/CashWithdrawalTxReceipt.kt

```diff
@@ -10,12 +10,13 @@ internal class CashWithdrawalTxReceipt : TxReceiptTemplate {
         merchant: Merchant?,
         terminalId: String,
         paymentMethod: PosPaymentMethod?,
-        receipt: Receipt
-    ) : super(merchant, terminalId, paymentMethod, receipt)
+        receipt: Receipt,
+        title: String
+    ) : super(merchant, terminalId, paymentMethod, receipt, title)
 
     override val txContent = CashWithdrawalLayout(
         snapBal,
         cashBal,
-        receipt.cashBackAmount
+        receipt.ebtCashAmount
```

Comment:
A withdrawal is indicated by a combo of `funding_type: "ebt_cash"` + `transaction_type: "withdrawal"` as best as I can tell from what is in [this linear issue](https://linear.app/joinforage/issue/FX-748/story-cash-withdrawal-flow), so we want to use the ebtCashAmount here.

Reply:
I think you're right on this one. We can confirm in the wet rehearsal tomorrow when all eyes are there but I vaguely remember that `cashBackAmount` did not always refer to the `withdrawal` amount or the `cashback` amount, which is what I mistakenly original thought the `cashBackAmount` meant. Nice catch!